gradle-plugins/src/integTest/groovy/com/linkedin/pegasus/gradle/PegasusPluginCacheabilityTest.groovy

```diff
@@ -96,6 +96,6 @@ class PegasusPluginCacheabilityTest extends Specification {
     preparedSchema.exists()
 
     where:
-    gradleVersion << [ '4.0', '5.2.1', '5.6.4', '6.9', '7.0.2' ]
+    gradleVersion << [ '5.2.1', '5.6.4', '6.9', '7.0.2' ]
```

Comment:
why moved `4.0`?

Reply:
Since we will be compiling RestLi in Java 11, we need to use Gradle 5.0 or higher, which is the minimum version required to support Java 11. As a result, I removed Gradle 4.0. This change was confirmed with @njuliyu from the Gradle Team, who added this test.